### PR TITLE
feat(core): deploy script for og mastercopy

### PR DESCRIPTION
**Pull request by Reinis-FRP**
**Motivation**

Automate deployment of Optimistic Governor mastercopy.

**Summary**

Adds deployment script for Optimistic Governor mastercopy.

**Details**

It is not intended to use deployed contract directly. Instead, users should deploy minimal proxy contracts that delegate calls to this mastercopy.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [x]  Untested

### CODE REVIEW

**Review by md0x — APPROVED** [review]
The script looks good!

**Review by mrice32 — APPROVED** [review]
LGTM!

File: packages/core/deploy/058_deploy_optimistic_governor.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// in its deployModule method's initializer parameter to call setUp on the deployed proxy contract.
+
+const func = async function (hre) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  const Finder = await deployments.get("Finder");
+  const owner = "0x000000000000000000000000000000000000dEaD"; // Mastercopy contract should not be usable directly.
```
Comment (chrismaree) [inline packages/core/deploy/058_deploy_optimistic_governor.js:17]: is there a reason it is this address and not the `0x0` address?

Comment (Reinis-FRP) [reply packages/core/deploy/058_deploy_optimistic_governor.js:17]: `transferOwnership` that is being called as part of constructor requires this to be non-zero address

**Pull request merged**